sys-devel/prelink/prelink-99999999.ebuild

```diff
@@ -52,6 +52,12 @@ src_install() {
 	exeinto /etc/cron.daily
 	newexe "${FILESDIR}"/prelink.cron prelink
 	newconfd "${FILESDIR}"/prelink.confd prelink
+
+	if use systemd; then
+		exeinto /usr/bin
+		newexe "${FILESDIR}"/prelink.cron systemd-prelink
```

Comment:
Renaming a fine is unfortunate here. I suggest using existing `/etc/cron.daily/prelink` file for timer unit and not introduce new script in `/usr/bin`. We can move it separately for both cron and time if need arises.